ksql-engine/src/main/java/io/confluent/ksql/util/AvroUtil.java

```diff
@@ -188,37 +188,32 @@ private static AbstractStreamCreateStatement addAvroFields(
     return abstractStreamCreateStatement.copyWith(elements, properties);
   }
 
-
-  public static void validatePersistentQueryResults(
+  public static boolean isValidSchemaEvolution(
```

Comment:
Changed this to return boolean, rather than throw, so that the calling code can throw a richer exception, (that includes the related statement text).